code/controllers/subsystem/research.dm

```diff
@@ -351,22 +357,30 @@ SUBSYSTEM_DEF(research)
 				result[i] *= eff
 				bitcoins[i] = bitcoins[i]? bitcoins[i] + result[i] : result[i]
 	else
-		for(var/obj/machinery/rnd/server/miner in servers)
+		for(var/obj/machinery/rnd/server/miner in BOSservers)
+			if(miner.working)
+				BOSbitcoins = BOSsingle_server_income.Copy()
+				//break
+		for(var/obj/machinery/rnd/server/miner in VAULTservers)
 			if(miner.working)
-				bitcoins = single_server_income.Copy()
-				break			//Just need one to work.
+				VAULTbitcoins = VAULTsingle_server_income.Copy()
+				//break	
```

Comment:
```suggestion
				break	
```
Why was this break removed? It literally just wastes processing time to remove it.